Palindrome/Palindrome/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace PalindromV
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.WriteLine("Enter a single word");
+			string input = Console.ReadLine();
+			string userInput = input.ToLower();
+
+			while (string.IsNullOrEmpty(input))
```

Comment:
Let's change it. Check that word consists of letters only. If it is wrong, give 3 attempts to enter the word (additional task for you)